docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
+        "admincredsaltmountname" which can be used with another Active
+        Directory server. The alternate name is illustrated in the example
+        below and should be added to your chart if necessary.</para>
+
+        <para>The "admincredsmountname" key/value pair already exists by
+        default in the HPCC Systems delivered values.yaml file. The key is
+        referenced in the component's ldap.yaml file. You may override these
+        and add additional key/values as needed.</para>
+
+        <programlisting lang="bash">  secrets:
+     authn:
+       admincredsmountname: "admincredssecretname"       #exernalize HPCC Admin creds
+       admincredsaltmountname: "admincredsaltsecretname" #exernalize alternate HPCC Admin creds
+</programlisting>
+      </sect3>
+
+      <sect3 id="ENABLE_k8S_LDAP">
+        <title>Enable LDAP Authentication</title>
+
+        <para>In the delivered
+        HPCC-Platform/esp/applications/common/ldap/ldap.yaml file, the
+        "ldapAdminSecretKey" is already set to the key mount name illustrated
+        in the example above. To enable LDAP authentication and to modify this
+        value, override the ESP/ECLWatch helm component located in values.yaml
+        as illustrated in the following example:</para>
+
+        <programlisting lang="YAML">esp:
+- name: eclwatch
+  application: eclwatch
+  auth: ldap
+  ldap:
+    ldapAddress: "myldapserver"
+    ldapAdminSecretKey: "admincredsaltmountname"   # use alternate secrets creds
+</programlisting>
+
+        <para> </para>
+      </sect3>
+    </sect2>
+
+    <sect2 id="Configuring_For_HashoCorpVault">
+      <title>Configuring LDAP in HashiCorp Vault</title>
+
+      <para>To create and store secrets in the HashiCorp Vault, from the
+      command command line interface, execute the following Vault commands.
+      The secret name used in the example below is "myvaultadmincreds" and
+      must be prefixed with "secret/authn/" as illustrated. The HPCC Systems
+      administrator "username" and "password" key/values are required; and
+      additional properties are ignored.</para>
+
+      <programlisting>vault kv put secret/authn/myvaultadmincreds username=hpcc_admin password=t0pS3cr3tP@ssw0rd </programlisting>
+
+      <para>This stores the values for the HPCC Systems administrator username
+      and password into the HashiCorp Vault.</para>
+
+      <para>To verify and confirm the secret values, execute the following
+      command:</para>
+
+      <programlisting lang="bash">vault kv get secret/authn/myvaultadmincreds</programlisting>
+
+      <para>For more information about creating secrets for HashiCorp Vault
+      see the appropriate HashiCorp documentation for your
+      implementation.</para>
+
+      <sect3 id="Deploying_HashiCorpVault">
+        <title id="DeployingTheHashiCorpVault">Deploying the HashiCorp
+        Vault</title>
+
+        <para>You can deploy the HashiCorp Vault secrets when you override the
```

Comment:
If using active voice, "You can deploy..." s/b "Deploy the HashiCorp Vault secrets by overriding the ..."